requirements.txt

```diff
@@ -0,0 +1 @@
+parsimonious>=0.10.0
```

Comment:
libraries should not have `requirements.txt` -- requirements.txt is for applications